network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/article_teaser_block.html

```diff
@@ -11,12 +11,10 @@
         the `pageurl` template tag would throw an error.
         {% endcomment %}
         {% if card.article %}
-          <div class="tw-px-4 tw-w-full medium:tw-w-1/3 tw-mb-5">
-            <div class="tw-bg-white tw-relative tw-py-4 tw-w-full tw-flex tw-flex-col">
-              <a href="{% pageurl card.article %}">
-                <span class="tw-h5-heading tw-mb-2">{{ card.article.title }}</span>
-              </a>
-            </div>
+          <div class="tw-w-full tw-pt-5 tw-border-t tw-border-black">
+            <a class="tw-h5-heading tw-block hover:tw-text-black " href="{% pageurl card.article %}">
```

Comment:
```suggestion
            <a class="tw-h5-heading tw-block hover:tw-text-black" href="{% pageurl card.article %}">
```
Oops ill remove this